nomad/rpc.go

```diff
@@ -306,6 +307,15 @@ func (s *Server) forwardLeader(server *serverParts, method string, args interfac
 	return s.connPool.RPC(s.config.Region, server.Addr, server.MajorVersion, method, args, reply)
 }
 
+// forwardServer is used to forward an RPC call to a particular server
+func (s *Server) forwardServer(server *serverParts, method string, args interface{}, reply interface{}) error {
+	// Handle a missing server
+	if server == nil {
```

Comment:
Looks like the nil case is addressed here rather than the caller, so my previous comment can be ignored